python/requirements.in

```
@@ -13,3 +13,4 @@ python-multipart  # Required for file uploads in FastAPI; see https://fastapi.ti
 shapely
 sgp4
 spacetrack
+git+https://github.com/nasa-gcn/python-dynamodb-autoincrement@v0.2.0
```

Use the release on PyPI, not the git tag:

```suggestion
dynamodb-autoincrement>=0.2.0
```

But please add this in alphabetical order.

---

Had no idea you'd released on PyPI, sorry. I've updated it.